conf/evolutions/default/104.sql

```diff
@@ -0,0 +1,11 @@
+# --- !Ups
+
+ALTER TABLE `patient_encounter_tab_fields`
+ADD COLUMN `IsDeleted` DATETIME NULL AFTER `chief_complaint_id`,
+ADD COLUMN `DeletedByUserId` INTEGER NULL AFTER `IsDeleted`;
```

Comment:
This needs to enforce a foreign key constraint on the users table for ID before we merge. This makes sure that the ID always links back to a user. Here's an example from the deleted_by_user_id in the patients table:

https://github.com/FEMR/femr/blob/master/conf/evolutions/default/88.sql